benchmark/BENCHMARK.md

```diff
@@ -38,20 +38,19 @@ Some scenarios (`CreateDeleteClients`, and `CrawlUsers`, `CreateRealms`, `Create
    * assign the roles `manage-clients` and `view-users`
 * the client secret to be passed to the tests can be copied from the `Credentials` tab
 
-#### Scenario `keycloak.scenario.admin.CreateRealms`
+#### Scenarii `keycloak.scenario.admin.CreateRealms` and `keycloak.scenario.admin.CreateDeleteRealms`
 
-This scenario requires the following Client settings, in addition to the requirements above:
+These scenarii are using the root admin account to perform realm operations with the built-in `admin-cli` client.
```

Comment:
a small change needed on `scenarii` to `scenarios`

Reply:
Done